### PR TITLE
Bump Application Registry and add timeout flags

### PR DESCRIPTION
**Pull request by franpog859**
**Description**

Changes proposed in this pull request:

- Bump image of Application Registry 
- Add `specRequestTimeout` to Application Registry values and deployment
- Add `assetstoreRequestTimeout` to Application Registry values and deployment

**Related issue(s)**

See the issue #4124 
See the implementation #4153

### CODE REVIEW

File: resources/application-connector/charts/application-registry/README.md [hunk 1]
```diff
@@ -12,4 +12,6 @@ This service has the following parameters:
 - **namespace** - The Namespace to which you deploy the Proxy Service. The default Namespace is `kyma-integration`.
 - **requestTimeout** - A time-out for requests sent through the Proxy Service. Provide it in seconds. The default time-out is `10`.
 - **requestLogging** - The flag to enable logging of incoming requests. The default value is `false`.
+- **specRequestTimeout** - A timeout for requests fetching specifications provided by the user. It is provided in seconds. The default timeout is `5`.
```
Comment (kazydek) [inline resources/application-connector/charts/application-registry/README.md]:
```suggestion
- **specRequestTimeout** - A time-out for requests fetching specifications provided by the user. It is provided in seconds. The default time-out is `5`.
```

File: resources/application-connector/charts/application-registry/README.md [hunk 2]
```diff
@@ -12,4 +12,6 @@ This service has the following parameters:
...
+- **assetstoreRequestTimeout** - A timeout for requests fetching specifications from the Asset Store Service. It is provided in seconds. The default timeout is `5`.
```
Comment (kazydek) [inline resources/application-connector/charts/application-registry/README.md]:
```suggestion
- **assetstoreRequestTimeout** - A time-out for requests fetching specifications from the Asset Store Service. It is provided in seconds. The default time-out is `5`.
```

File: resources/application-connector/charts/application-registry/README.md (hunk 1 above)
Comment (kazydek) [reply resources/application-connector/charts/application-registry/README.md]: --> As in [MSG](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/time-out) guidelines that we follow ;)

Comment (akgalwas) [reply resources/application-connector/charts/application-registry/README.md]: Done

File: resources/application-connector/charts/application-registry/README.md (hunk 2 above)
Comment (akgalwas) [reply resources/application-connector/charts/application-registry/README.md]: Done

**Pull request merged**